libs/net/entities/AdminConfigurableSettingNames.cs

```diff
@@ -0,0 +1,15 @@
+using TNO.Core.Json;
+
+namespace TNO.Entities;
+
+/// <summary>
+/// Provides report status.
+/// </summary>
+public enum AdminConfigurableSettingNames
```

Comment:
Confused how this is used?

Reply:
It’s used the same way as a string const would be. I just use a tostring to get the value to look up.

Reply:
Just noticed the summary is wrong from copy pasta